includes/definitions.inc.php

```diff
@@ -1577,6 +1577,14 @@
 $config['os'][$os]['over'][0]['graph'] = 'device_bits';
 $config['os'][$os]['over'][0]['text']  = 'Traffic';
 
+// HP Virtual Connect
+$os = 'hpvc';
```

Comment:
HP does not have any (public) OS name for the software the modules are running, so I went with an acronym of HP Virtual Connect to keep it short.